usr/sbin/p11sak/p11sak.c

```diff
@@ -2254,6 +2248,7 @@ static CK_RV generate_asymmetric_key(CK_SESSION_HANDLE session, CK_SLOT_ID slot,
     CK_MECHANISM mech;
     CK_ULONG i;
     CK_RV rc;
+    char separator = ':';
```

Comment:
Here you use `:`, man page says `|`.  Also, why not `const`?

Reply:
Done